fmn/sse/subscriber.py

```diff
@@ -44,23 +51,38 @@ def write_messages_all_connections(self, key):
         if payload:
             self.logger.info(payload)
             connections = self.get_connections(key=key)
+            sse_msg = self.format_msg_sse(msg=payload)
             for req in connections:
-                self.logger.info(req)
-                self.push_sse(payload, req)
+                self.push_sse(sse_msg, req)
+            self.logger.info("Total Connections: " + str(len(connections)))
+
+    def get_feedqueue(self, key):
```

Comment:
It would be good to include a docblock.

This is outside the scope of this PR, but a common pattern (and one that I think we should adopt here) is to use a ``namedtuple`` rather than a plain tuple or list. This _greatly_ increases readability.